** Backup-manager's version string includes "-debian"
   We want people to know that it differs from upstream.

   Provided by: Sven Joachim <svenjoac@gmx.de>


---
 backup-manager |    3 +++
 1 file changed, 3 insertions(+)

--- a/backup-manager
+++ b/backup-manager
@@ -84,6 +84,9 @@
     VERSION="${VERSION}r${rev}"
 fi
 
+# Append "-debian" for Debian versions
+VERSION="${VERSION}-debian"
+
 debug "Version : $VERSION, release : $RELEASE"
 
 # set useful global variables and initial
